middleware/kubernetes/README.md

```diff
@@ -1,88 +1,126 @@
+	#
+	# Only expose the k8s namespaces listed.  If this option is omitted
+	# all namespaces are exposed
+	#
+	namespaces demo
+	
+	# lables <expression> [,<expression>] ...
+	#
+	# Only expose the records for kubernetes objects
+	# that match this label selector. The label
+	# selector syntax is described in the kubernetes
+	# API documentation: http://kubernetes.io/docs/user-guide/labels/
+	# Example selector below only exposes objects tagged as
+	# "application=nginx" in the staging or qa environments.
+	#
+	labels environment in (staging, qa),application=nginx
+	
+	# pods <disabled|insecure|verified>
+	#
+	# Set the mode of responding to pod A record requests. 
+	# e.g 1-2-3-4.ns.pod.zone.  This option is provided to allow use of
+	# SSL certs when connecting directly to pods.
+	# Valid values: disabled, verified, insecure
+	#  disabled: Do not process pod requests, always returning NXDOMAIN
+	#  insecure: Always return an A record with IP from request (without 
+	#            checking k8s).  This option is is vulnerable to abuse if
+	#            used maliciously in conjuction with wildcard SSL certs.
+	#  verified: Return an A record if there exists a pod in same 
+	#            namespace with matching IP.
+	# Default value is "disabled".
+	#
 	pods disabled
-    }
-    # Perform DNS response caching for the coredns.local zone
-    # Cache timeout is specified by an integer in seconds
-    #cache 180 coredns.local
+	
+	# cidrs <cidr> [<cidr>] ...
+	#
+	# Expose cidr ranges to reverse lookups.  Include any number of space
+	# delimited cidrs, and or multiple cidrs options on separate lines.
+	# kubernetes middleware will respond to PTR requests for ip addresses
+	# that fall within these ranges.
+	#
+	cidrs 10.0.0.0/24 10.0.10.0/25
+		
 }
-~~~
 
-Defaults:
-* If the `namespaces` keyword is omitted, all kubernetes namespaces are exposed.
-* If the `resyncperiod` keyword is omitted, the default resync period is 5 minutes.
-* The `labels` keyword is only used when filtering results based on kubernetes label selector syntax
-  is required. The label selector syntax is described in the kubernetes API documentation at:
-  http://kubernetes.io/docs/user-guide/labels/
-* If the `pods` keyword is omitted, all pod type requests will result in NXDOMAIN
+```
+
+### Wildcards
+
+Some query labels accept a wildcard value to match any value. 
+If a label is a valid wildcard (*, or the word "any"), then that label will match 
+all values.  The labels that accept wildcards are:
+* _service_ in an `A` record request: _service_.namespace.svc.zone.
+   * e.g. *.ns.scv.myzone.local
+* _namespace_ in an `A` record request: service._namespace_.svc.zone.
+   * e.g. nginx.*.svc.myzone.local
+* _port and/or protocol_ in an `SRV` request: __port_.__protocol_.service.namespace.svc.zone.
+   * e.g. \_http.*.service.ns.svc.
+* multiple wild cards are allowed in a single query.
+   * e.g. `A` Request *.*.svc.zone. or `SRV` request *.*.*.*.svc.zone.
```

Comment:
These `*s` don't come through in the rendered markdown. Putting them in backquotes is the easiest way to fix that.